sdk/python/feast/feature_view.py

```diff
@@ -169,7 +169,7 @@ def to_proto(self) -> FeatureViewProto:
             ttl_duration = Duration()
             ttl_duration.FromTimedelta(self.ttl)
 
-        print(f"Stream soruce: {self.stream_source}, {type(self.stream_source)}")
+        print(f"Stream source: {self.stream_source}, {type(self.stream_source)}")
```

Comment:
I actually think we can delete this line altogether!

Reply:
(i think I left it in while running/debugging locally)

Reply:
Haha - not a problem. This was when I was testing the offline config.